### PR TITLE
chore(11.0.0): prepare new release

### PR DESCRIPTION
**Pull request by robertohuertasm**

### CODE REVIEW

**Comment by JimiC** [comment]
@robertohuertasm You can combine #2563 and #2614. Change the release date as well. Other than that LGTM.

**Review by KingDarBoja — DISMISSED** [review]
Everything is good! 🚀

**Review by JimiC — CHANGES_REQUESTED** [review]
We said we do a major version update.

**Comment by robertohuertasm** [comment]
Ready!

**Comment by robertohuertasm** [comment]
2 last minute PRs added to the changelog!

**Review by JimiC — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # Change Log
 
+## 11.0.0 (September 17, 2020)
+
+- Enhancement: Support for `cjs/json5` for `prettier`. ([@shrujalshah28](https://github.com/**mvshmakov**) in [#2612](https://github.com/vscode-icons/vscode-icons/pull/2612))
```
Comment (JimiC) [inline CHANGELOG.md:5]: REmove asterisks

**Review by JimiC — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JimiC) [reply CHANGELOG.md:5]: The correct link is `https://github.com/shrujalshah28`

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 - Feature: Support for `BlitzBasic`. ([@Lukaszeq](https://github.com/Lukaszeq) in [#2609](https://github.com/vscode-icons/vscode-icons/pull/2609))
 - Enhancement: Add `BUILD.bazel` extension and update language id. ([@CaringDev](https://github.com/CaringDev) in [#2605](https://github.com/vscode-icons/vscode-icons/pull/2605))
 - Feature: Support for `Hook(s)` folder. ([@chancegraff](https://github.com/chancegraff) in [#2604](https://github.com/vscode-icons/vscode-icons/pull/2604))
+- Enhancement: Add `cjs` to `rollup`. ([@sant123](https://github.com/sant12) in [#2603](https://github.com/vscode-icons/vscode-icons/pull/2603))
```
Comment (JimiC) [inline CHANGELOG.md:11]: The correct link is `https://github.com/sant123`

File: CHANGELOG.md (hunk 1 above)
Comment (robertohuertasm) [reply CHANGELOG.md]: solved

File: CHANGELOG.md (hunk 2 above)
Comment (robertohuertasm) [reply CHANGELOG.md]: solved

**Review by JimiC — APPROVED** [review]

**Comment by robertohuertasm** [comment]
Thanks for catching the errors! Not sure what happened with those asterisks. I thought I removed them.

**Pull request merged**